### PR TITLE
Cleanup setup instructions

### PR DESCRIPTION
**Pull request by sduckett**
We have Nightcode rather than Light Table
Screenshots updated, so you should be able to follow along and verify that you're on track.

### CODE REVIEW

File: outline/setup_osx.md [hunk 1]
```diff
@@ -86,32 +83,17 @@ Now run `which lein` and you should see the `lein` command.
 
 After you set up Leiningen as above, run the `lein version` command. This should take a while to run, as it will download some resources it needs the first time. If it completes successfully, you are golden! If not, ask an instructor for help.
 
-## Installing Light Table
+## Installing Nightcode
 
-Go to the [LightTable site](http://www.lighttable.com/). On the page there, you should see a set of buttons that have download links for Light Table. Click the "OS X 10.7+" button and you will download a .zip file.
+Go to the [Nightcode site](https://sekao.net/nightcode/). On the page there, click the "Free Download" link and you will download a .jar file. There should now be a file named `nightcode-0.4.7-standalone.jar` in your Downloads folder. If Java is installed, you can double click the JAR file to run it. Depending on your Mac's security settings, you may see an error message that says the author of the file can't be verified. You'll be able to run Nightcode if you relax the security settings.
```
Comment (iantruslove) [inline outline/setup_osx.md:88]: (Some/most/all) OSX users will have the option of getting Nightcode from the App store. Their setup should be simpler, but we'll need to repeat this good work for that install flow.

File: outline/setup_osx.md [hunk 2]
```diff
@@ -135,46 +117,24 @@ This will put you in the directory with the source code for this sample bit of C
...
+![Testing lein repl](img/testing-step2.png)
 
-Now, start Applications > LightTable. Once it is started, press the Control button and Space Bar together (abbreviated Ctrl+Space). This is how you start giving LightTable a command. Start typing the word "instarepl" and you should see a menu of options, like below. Choose "Instarepl: open a clojure instarepl."
+This is starting a REPL, which we will learn about soon. It's a special terminal for Clojure. At the REPL prompt, type `(+ 1 1)` and press Return. Did you get the answer `2` back? You will learn more about that in the course. For now, press the Control button and D button on your keyboard together (abbreviated as Ctrl+D). This should take you out of the Clojure REPL and back to your normal terminal prompt. Then, the terminal will show you the following message: `user=> Bye for now!`
```
Comment (iantruslove) [inline outline/setup_osx.md]: Suggest: "d button" instead of "D button", and "Ctrl+d" instead of "Ctrl+D"

File: outline/setup_ubuntu.md [hunk 3]
```diff
@@ -54,38 +54,13 @@ export PATH=$PATH:/usr/local/bin
 
 After you run the above commands, run the `lein version` command. It should take a while to run, as it will download some resources it needs the first time. If it completes successfully, you are golden! If not, ask an instructor for help.
 
-## Installing Light Table
+## Installing Nightcode
 
-You will need to know whether you are running the 32-bit or 64-bit version of Ubuntu. To find out, click Dash Home and type Details. You should see a window like this:
+Go to the [Nightcode site](https://sekao.net/nightcode/). On the page there, click the "Free Download" link and you will download a .jar file. There should now be a file named `nightcode-0.4.7-standalone.jar` in your Downloads folder. If Java is installed, you can double click the JAR file to run it. Depending on your Mac's security settings, you may see an error message that says the author of the file can't be verified. You'll be able to run Nightcode if you relax the security settings.
```
Comment (iantruslove) [inline outline/setup_ubuntu.md]: Delete the last 2 sentences - they're OSX-specific

File: outline/setup_win7.md [hunk 4]
```diff
@@ -49,16 +49,13 @@ Leiningen is a tool used on the command line to manage Clojure projects.
...
 
-## Installing Light Table
+## Installing Nightcode
 
-Go to the [Light Table site](http://www.lighttable.com/). On the page there, you should see a set of buttons that have download links for Light Table. Click the "Win" button and you will download a .zip file.
+Go to the [Nightcode site](https://sekao.net/nightcode/). On the page there, click the "Free Download" link and you will download a .jar file. There should now be a file named `nightcode-0.4.7-standalone.jar` in your Downloads folder. If Java is installed, you can double click the JAR file to run it. Depending on your Mac's security settings, you may see an error message that says the author of the file can't be verified. You'll be able to run Nightcode if you relax the security settings.
```
Comment (iantruslove) [inline outline/setup_win7.md]: Same as before - there are Mac instructions in this line

File: outline/setup_win7.md [hunk 5]
```diff
@@ -103,77 +100,60 @@ Expected result:
...
 
-![Testing lein repl](img/win7/testing-step2.png)
+![Testing lein repl](img/testing-step2.png)
 
-This is starting a REPL, which we will learn about soon. It's a special terminal for Clojure. At the REPL prompt, type `(+  1  1)` and hit enter. Did you get the answer `2` back? You will learn more about that in the course. For now, press the Control button and D button on your keyboard together (abbreviated as Ctrl+D). This should take you out of the Clojure REPL and back to your normal terminal prompt.
+This is starting a REPL, which we will learn about soon. It's a special terminal for Clojure. At the REPL prompt, type `(+  1  1)` and press Return. Did you get the answer `2` back? You will learn more about that in the course. For now, press the Control button and D button on your keyboard together (abbreviated as Ctrl+D). This should take you out of the Clojure REPL and back to your normal terminal prompt. Then, the terminal will show you the following message: `user=> Bye for now!`
```
Comment (iantruslove) [inline outline/setup_win7.md]: Question not a comment: is it still Ctrl-D on a Windows machine?

File: outline/setup_ubuntu.md (hunk 3 above)
Comment (sduckett) [inline outline/setup_ubuntu.md]: Good catch.

File: outline/setup_osx.md (hunk 2 above)
Comment (sduckett) [inline outline/setup_osx.md]: No need to imply there's a shift, I suppose. That said, it's 'D' on my keyboard. :musical_keyboard:

File: outline/setup_osx.md (hunk 1 above)
Comment (sduckett) [inline outline/setup_osx.md:88]: :cake:

File: outline/setup_win7.md (hunk 4 above)
Comment (sduckett) [inline outline/setup_win7.md]: :+1:

File: outline/setup_win7.md (hunk 5 above)
Comment (sduckett) [inline outline/setup_win7.md]: I'll get back to you on this.

**Comment by sduckett** [comment]
We should add some notes about the app store workflow once we try it out.

**Comment by iantruslove** [comment]
+1

**Comment by westzimm** [comment]
inc! Thank you, Sean!!

**Pull request merged**